lib/jxl/enc_fast_lossless.h

```diff
@@ -50,6 +50,11 @@ void JxlFastLosslessPrepareHeader(JxlFastLosslessFrameState* frame,
 // JxlFastLosslessPrepareHeader.
 size_t JxlFastLosslessMaxRequiredOutput(const JxlFastLosslessFrameState* frame);
 
+// Actual size of the frame once it is encoded. This is not identical to
+// JxlFastLosslessOutputSize because JxlFastLosslessWriteOutput may require
```

Comment:
I think you meant "not identical to JxlFastLosslessMaxRequiredOutput"

Reply:
haha, yep!